skimage/data/tests/test_data.py

```diff
@@ -102,6 +102,9 @@ def test_lfw_subset():
 
 
 def test_cell():
-    """ Test that "cell" image can be loaded. """
+    """ Test that "cell" image can be loaded."""
+    # Needs pooch, or git development. using testing.fetch will
+    # cause the test to be skipped instead of failing if neither of
+    # those two conditions are met.
+    fetch('data/cell.png')
```

Comment:
Why fetch this image but not others?

Reply:
This fetch should catch the local version, no?